instantsearch-compose/src/main/java/com/algolia/instantsearch/compose/filter/internal/FacetListConnectionPager.kt

```diff
@@ -0,0 +1,29 @@
+package com.algolia.instantsearch.compose.filter.internal
+
+import com.algolia.instantsearch.compose.paging.SearcherPager
+import com.algolia.instantsearch.core.connection.ConnectionImpl
+import com.algolia.instantsearch.helper.filter.facet.FacetListViewModel
+import com.algolia.search.model.search.Facet
+
+/**
+ * Connection implementation between [FacetListViewModel] and [SearcherPager].
+ */
+internal class FacetListConnectionPager<T : Any>(
+    private val facetListViewModel: FacetListViewModel,
+    private val pager: SearcherPager<T>
+) : ConnectionImpl() {
+
+    private val callback: (List<Pair<Facet, Boolean>>) -> Unit = {
```

Comment:
Even though this variable is private, maybe it makes sense to give it a more meaningful name?